tests/src/test/scala/com/nvidia/spark/rapids/AnsiCastOpSuite.scala

```diff
@@ -564,7 +561,9 @@ class AnsiCastOpSuite extends GpuExpressionTestSuite {
     checkCastWithParseError("2015.03.18")
     checkCastWithParseError("20150318")
     checkCastWithParseError("2015-031-8")
-    checkCastWithParseError("2015-03-18T12:03:17-0:70")
+
+// TODO: add before merge!!!!, it's a tempt remove. It's for test pre-merge
+//    checkCastWithParseError("2015-03-18T12:03:17-0:70")
```

Comment:
Will revert this afater JNI snapshot jar with fix (https://github.com/NVIDIA/spark-rapids-jni/pull/3394/) is ready

Reply:
Reverted  due to JNI repo already fixed it